storage/storage_src.go

```diff
@@ -136,8 +137,10 @@ func (s *storageImageSource) GetBlob(ctx context.Context, info types.BlobInfo, c
 	}()
 	// On Unix and modern Windows (2022 at least) we can eagerly unlink the file to ensure it's automatically
 	// cleaned up on process termination (or if the caller forgets to invoke Close())
+	// On older versions of Windows we will have to fallback to relying on the caller to invoke Close()
+	removeFailure := false
 	if err := os.Remove(tmpFile.Name()); err != nil {
```

Comment:
(Should this check for a specific error condition? I’m not sure; I guess I/O errors would be reported in the `io.Copy` afterwards… OTOH we don’t want to add too many platform conditionals.)

Reply:
I thought about this, but as you said it adds a platform based condition. I can but something in if you think it will be beneficial